packages/core/src/progress-circle/progress-circle.stories.ts

```diff
@@ -0,0 +1,199 @@
+  return html` <cds-progress-circle ...="${spreadProps(getElementStorybookArgs(args))}"></cds-progress-circle> `;
+};
+
+export const spinner = () => {
+  return html`
+    <cds-progress-circle
+      size="xl"
+      aria-label="Demonstration of indeterminate progress circle spinning to show work is in progress"
+    ></cds-progress-circle>
+  `;
+};
+
+export const status = () => {
+  const labelText = 'Demonstration of circular progress ';
+  return html`
+    <div cds-layout="horizontal gap:sm">
+      <cds-progress-circle
+        size="xl"
+        value="0"
+        role="progressbar"
+        aria-valuenow="0"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        value="12"
+        role="progressbar"
+        aria-valuenow="12"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        value="100"
+        role="progressbar"
+        aria-valuenow="100"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        status="info"
+        value="48"
+        role="progressbar"
+        aria-valuenow="48"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        status="success"
+        value="72"
+        role="progressbar"
+        aria-valuenow="72"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        status="warning"
+        value="36"
+        role="progressbar"
+        aria-valuenow="36"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        status="danger"
+        value="84"
+        role="progressbar"
+        aria-valuenow="84"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <cds-progress-circle
+        size="xl"
+        status="unknown"
+        value="60"
+        role="progressbar"
+        aria-valuenow="60"
+        aria-valuemin="0"
+        aria-valuemax="100"
+      ></cds-progress-circle>
+      <div style="background: #313131">
+        <cds-progress-circle
+          size="xl"
+          inverse
+          value="24"
+          role="progressbar"
+          aria-valuenow="24"
+          aria-valuemin="0"
+          aria-valuemax="100"
+        ></cds-progress-circle>
+      </div>
+    </div>
+  `;
+};
+
+// this is kept here so we can verify the animation when we need to.
+// to test in screen reader, need to have a long interval.
+// note for future dev, throttle dynamic progress updates so as not to spam the screen reader.
+export const chaos = () => {
+  return html`
+    <span id="progress-announce" cds-layout="display:screen-reader-only" aria-live="polite"
```

Comment:
This is a good example for product teams needing to announce progress in their applications. Like the other examples, it came from level access.